flink-python/pyflink/pyflink_gateway_server.py

```diff
@@ -41,8 +41,9 @@ def find_java_executable():
     flink_conf_path = os.path.join(flink_home, "conf", "flink-conf.yaml")
     java_home = None
 
-    if os.path.isfile(flink_conf_path):
-        with open(flink_conf_path, "r") as f:
+    real_flink_conf_path = os.path.realpath(flink_conf_path)
```

Comment:
Maybe add some comments here that we use realpath to avoid CWE22 problems?

Reply:
Thanks you for your comments! Yes, it's better to add some comments to be more readable.